React/Views/RefreshControl/RCTRefreshControl.m

```diff
@@ -160,6 +163,11 @@ - (void)setCurrentRefreshingState:(BOOL)refreshing
   _currentRefreshingStateTimestamp = _currentRefreshingStateClock++;
 }
 
+- (void)setProgressViewOffset:(float)offset
+{
+	_progressViewOffset = offset / UIScreen.mainScreen.scale;
```

Comment:
why is this needed? Isn't `offset` already in points?

Reply:
Without the scaling, the offset using `CGRectOffset` didn't match up with the android `progressViewOffset`